mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -17,8 +16,9 @@
 from ..shared.timekeeping.utility import get_simulation_start_time, \
```

Comment:
yes, it is, but I couldn't leave the comment in the right place.. It is referring to the lines right above this one, where `open_multifile_dataset` is imported. sorry.